app.js

```diff
@@ -1,14 +1,22 @@
 const express = require('express')
 const app = express()
-const get = express()
 app.use(express.json())
-
-const {getCategories, getReviews} = require('./controller')
+const {getCategories, getReviews, getReviewById} = require('./controller')
 
 app.get('/api/categories', getCategories)
 app.get('/api/reviews', getReviews)
+app.get('/api/reviews/:review_id', getReviewById)
 
-
+app.use((err, request, response, next) =>{
+    if(err.status&&err.msg){
+        response.status(err.status).send({msg: err.msg})
+    }next(err)
+})
+app.use((err, request, response, next) =>{
+    if(err.code === '22P02'){
+        response.status(400).send({msg: 'Bad Request!'})
+    }
+})
```

Comment:
You've the `next` parameter available to you, so worth thinking for your next ticket what should come next in this chain